context/auth-context.tsx

```diff
@@ -163,9 +187,12 @@ export const AuthProvider = ({ children }: { children: ReactNode }) => {
     isProtectedRoute,
     authResponse,
     activeWalletChanged,
+    activeChainChanged,
+    setActiveChainChanged,
   } = useAuthProvider();
 
   const userStore = useUserStore();
+  const switchChain = useSwitchActiveWalletChain();
```

Comment:
this seems to be duplicate. This statement appears in line 57 as well.
Also this this hook seems unused in auth-context.tsx.